templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -13,6 +13,10 @@ import (
 <%
 	# {Compute}{Address}_{addressBasic}
 	test_slug = "#{product}#{resource_name}_#{example.name.camelize}Example"
+	ignore_read = data[:object].all_user_properties
+														 .select(&:ignore_read)
```

Comment:
be careful about tabs

Reply:
Oh tabstops are hard. Mine was set to 2 locally and this is using 4 on Github. I've changed this to have the right # for 4.